### DISCUSSION TITLE
[Question] Publishing

### DISCUSSION THREAD

**Discussion by ErikEJ** [Giorgi/EFCore.Visualizer#7] [General]
Previously you were not allowed to publish extensions using the new extensibility SDK - has this restriction been lifted?

**Comment by Giorgi** [comment]
No, as far as I know, that restriction is still in place but this project is built on a template with VSSDK compatibility and that template allows building and publishing VSIX.

Check out [How to publish extensions built with VSExtensibility](https://github.com/microsoft/VSExtensibility/issues/233)
